cgi-bin/DW/Controller/Export.pm

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/perl
+#
+# DW::Controller::Export
+#
+# Pages for exporting journal content.
+#
+# Authors:
+#      Mark Smith <mark@dreamwidth.org>
+#
+# Copyright (c) 2015 by Dreamwidth Studios, LLC.
+#
+# This program is free software; you may redistribute it and/or modify it under
+# the same terms as Perl itself. For a copy of the license, please reference
+# 'perldoc perlartistic' or 'perldoc perlgpl'.
+#
+
+package DW::Controller::Export;
+
+use v5.10;
+use strict;
+use warnings;
```

Comment:
We don't actually use warnings on a per-file level; it's on for dev servers and off for prod.